config/datasets/HEAD-NECK-RADIOMICS-HN1.yaml

```
@@ -0,0 +1,22 @@
+OUTCOME_VARIABLES:
+    time_label: "overall_survival_in_days"
+    event_label: "event_overall_survival"
+    convert_to_years: True
+    event_value_mapping: {}
+
+EXCLUSION_VARIABLES: {}
+
+TRAIN_TEST_SPLIT:
+    split: False
+    split_variable: {}
+    impute:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)